### PR TITLE
{chem}[intel/2021a] OpenMolcas 21.06

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: easybuild/easyconfigs/g/GlobalArrays/GlobalArrays-5.8-intel-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+configopts = ' --with-mpi --enable-i8'
+configopts += ' --with-blas8="-L$EBROOTIMKL/mkl/latest/lib/intel64 -lmkl_sequential -lmkl_intel_ilp64"'
+configopts += ' --with-scalapack8="L${MKLROOT}/lib/intel64 -lmkl_scalapack_ilp64 -lmkl_intel_ilp64 '
+configopts += '-lmkl_sequential -lmkl_core -lmkl_blacs_intelmpi_ilp64 -lpthread -lm -ldl"'
+
+# select armci network as (Comex) MPI-1 two-sided
+configopts += ' --with-mpi-ts'
+
+# use IB network
```
Comment (smoors) [inline easybuild/easyconfigs/g/GlobalArrays/GlobalArrays-5.8-intel-2021a.eb:24]: @hajgato this comment looks out of place?

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
node309.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/00c54bb807c52257a4b4549d07a084ca for a full test report.

File: easybuild/easyconfigs/g/GlobalArrays/GlobalArrays-5.8-intel-2021a.eb (hunk 1 above)
Comment (hajgato) [reply easybuild/easyconfigs/g/GlobalArrays/GlobalArrays-5.8-intel-2021a.eb]: done

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 7 out of 7 (2 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/b14c7c2eae2a4c3173b56d8e586f09de for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — DISMISSED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3109.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/d5e3aaff3a976827ff9774d3af13dc35 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3510.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/457f71e083bbe15362764d0fbc58caa8 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**